server/src/DBTablesMonitoring.cc

```diff
@@ -998,6 +999,25 @@ string EventsQueryOption::getCondition(void) const
 		statusCondition = string("(") + statusCondition + string(")");
 	addCondition(condition, statusCondition);
 
+	string incidentStatusCondition;
+	for (auto status: m_impl->incidentStatuses) {
```

Comment:
We should use reference for 'status' to suppress a copy of string objects.
auto status -> auto &status